### ISSUE TITLE
Question about training

### ISSUE THREAD

**Issue by XinzeShen** [MenghaoGuo/PCT#16]
Hello Guo! I found that much more epochs were needed in part_seg task when I replaced the model in Pointnet++ with PCT. (Pointnet++ code from https://github.com/yanx27/Pointnet_Pointnet2_pytorch) 

Pointnet++ needs less than 50 epochs while PCT needs more than 500 epochs. Is this caused by model complexity? Have you found same question when training?

**Comment by MenghaoGuo** [comment]
We don't observe this in the experiments. I suggest you can adjust this initial learning rate.
